sites/main-site/src/content/events/2024/hackathon-barcelona.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Hackathon - May 2024 (Barcelona)
+subtitle: An in-person hackathon held in Barcelona
+type: hackathon
+announcement:
+    start: 2024-10-28T09:00:00+02:00
```

Comment:
are you sure you don't want to announce it earlier?

Reply:
Noted and fixed - good shout!